capture/capture.yaml

```diff
@@ -83,16 +83,23 @@ components:
             The byte offset to filter on
           type: integer
           default: 0
+        bit_length:
+          description: >-
+            The bit length to filter on
+          type: integer
+          default: 8
     
     Capture.Field:
       type: object
       properties:
         value:
           type: string
           format: hex
+          default: 0
```

Comment:
should the defaults be "0" or 0 ?

Reply:
In the other fields like mac and ip default was mentioned without quotation. That's why I didn't use quotation.

Reply:
updated